source/reference/methods.txt

```diff
@@ -251,10 +251,99 @@ Collection Methods
 
      - Modifies multiple documents in a collection.
 
+Bulk Operation Methods
+----------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+
+   * - :method:`db.collection.initializeOrderedBulkOp()`
```

Comment:
Should these go after the `Bulk...` methods if we're going for alphabetical?

Reply:
I used the same ordering that they do in the [manual](https://docs.mongodb.com/manual/reference/method/js-bulk/). I would imagine it's because all the other methods are basically sub-methods to those top two; you can't use the sub-methods without first calling one of the top-level methods.